great-idea/index.html

```diff
@@ -17,7 +17,87 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
+           
+    <div class="navigation">  
+      <nav>
+        <a href="#">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+           <img class="logo" src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/logo.png" alt="Great Idea! Company logo.">
+   </div>
+   
+   <header>
+     <div class="header">
+    <div class="demand">
+     <h1> Innovation<br>
+     On<br>
+     Demand<br>
```

Comment:
Good job catching the easiest method, using break tags. You could have also done this with span tags and then applying custom CSS properties to that tag.